src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -679,33 +725,49 @@ class BasicSettings extends React.Component {
         {/* Advanced CPU Topology Options */}
         <ExpandCollapse id={`${idPrefix}-advanced-options`} textCollapsed={msg.advancedCpuTopologyOptions()} textExpanded={msg.advancedCpuTopologyOptions()}>
           <Grid className={style['settings-container']}>
-            <FieldRow fieldCols={3} label={msg.virtualSockets()} id={`${idPrefix}-topology-sockets`}>
+            <FieldRow
+              fieldCols={3}
+              label={msg.virtualSockets()}
+              id={`${idPrefix}-topology-sockets`}
+            >
               <SelectBox
                 id={`${idPrefix}-topology-sockets-edit`}
                 items={this.mapVCpuTopologyItems(vCpuTopologyDividers.sockets)}
                 selected={data.topology.sockets.toString()}
                 onChange={selectedId => { this.handleChange('topology', selectedId, { vcpu: 'sockets' }) }}
+                disabled={indicators.cpu}
```

Comment:
```suggestion
                disabled={!!indicators.cpu}
```

This is causing an error on browser console:
> Warning: Failed prop type: Invalid prop `disabled` of type `string` supplied to `SelectBox`, expected `boolean`.